tests/proxy_models/tests.py

```diff
@@ -188,6 +189,19 @@ def test_permissions_created(self):
         from django.contrib.auth.models import Permission
         Permission.objects.get(name="May display users information")
 
+    def test_permission_with_correct_content_type_created(self):
+        from django.contrib.auth.models import Permission
+        opts = AuthUserProxy._meta
+        codename = get_permission_codename('add', opts)
+        try:
```

Comment:
try/except/fail is generally considered an antipattern, e.g. 7fec264e46d2a757f06f857e513072d72686cf9d. Maybe it's okay here.